nvim/child_process_stream.lua

```diff
@@ -16,7 +16,7 @@ function ChildProcessStream.spawn(argv, env)
     args[#args + 1] = argv[i]
   end
   self._proc, self._pid = uv.spawn(prog, {
-    stdio = {self._child_stdin, self._child_stdout, 2},
+    stdio = {self._child_stdin, self._child_stdout, 2, io_extra},
```

Comment:
is this a hidden undocumented lua-client feature? I don't see this in Luv docs: https://github.com/luvit/luv/blob/master/docs.md